apitest/scripts/rolling-offer-simulation.sh

```diff
@@ -0,0 +1,119 @@
+#! /bin/bash
+
+# Demonstrates a way to always keep one offer in the market, using the API CLI with a local regtest bitcoin node.
+# Alice creates an offer, waits for Bob to take it, and completes the trade protocol with him.  Then Alice
+# creates a new offer...
+#
+# Stop the script by entering ^C.
+#
+# A country code argument is used to create a country based face to face payment account for the simulated offer.
+#
+# Prerequisites:
+#
+#  - Linux or OSX with bash, Java 10, or Java 11-12 (JDK language compatibility 10), and bitcoin-core (v0.19, v0.20).
+#
+#  - Bisq must be fully built with apitest dao setup files installed.
+#    Build command:  `./gradlew clean build :apitest:installDaoSetup`
+#
+#  - All supporting nodes must be run locally, in dev/dao/regtest mode:
+#           bitcoind, seednode, arbdaemon, alicedaemon, bobdaemon
+#
+#    These should be run using the apitest harness.  From the root project dir, run:
+#    `$ ./bisq-apitest --apiPassword=xyz --supportingApps=bitcoind,seednode,arbdaemon,alicedaemon,bobdaemon --shutdownAfterTests=false`
```

Comment:
This kind of simulation would be nice to have for normal regtest that I would run. Could you add instructions on how to run this for my normal alice client without starting the other as I already have them started? I just want a script that uses my normal regtest alice data dir which is already funded and has a history of trades, failed trades and other weird history.

Reply:
This is related to https://github.com/bisq-network/bisq/pull/5129#discussion_r569491295.   We had a chat on keybase and agreed that dev-test script or java testcases will be added, and these scripts' scope will be limited to serving as non-interactive  demos / tutorials.